app-backend/migrations/src_migrations/main/scala/54.scala

```diff
@@ -0,0 +1,38 @@
+import slick.driver.PostgresDriver.api._
+import com.liyaos.forklift.slick.SqlMigration
+
+object M54 {
+  RFMigrations.migrations = RFMigrations.migrations :+ SqlMigration(54)(List(
+
+    sqlu"""
```

Comment:
I don't know, but @notthatbreezy didn't object to that when I asked in the main channel. Notice the parent structure is a `List`.